src/ray/stats/stats.h

```diff
@@ -90,17 +90,12 @@ static inline void Init(const TagsType &global_tags,
   // Set interval.
   StatsConfig::instance().SetReportInterval(absl::Milliseconds(std::max(
       RayConfig::instance().metrics_report_interval_ms(), static_cast<uint64_t>(1000))));
-  StatsConfig::instance().SetHarvestInterval(
-      absl::Milliseconds(std::max(RayConfig::instance().metrics_report_interval_ms() / 2,
-                                  static_cast<uint64_t>(500))));
 
   MetricPointExporter::Register(exporter, metrics_report_batch_size);
   OpenCensusProtoExporter::Register(
       metrics_agent_port, (*metrics_io_service), "127.0.0.1");
   opencensus::stats::StatsExporter::SetInterval(
       StatsConfig::instance().GetReportInterval());
-  opencensus::stats::DeltaProducer::Get()->SetHarvestInterval(
```

Comment:
Hmm maybe we shouldn't remove this (maybe 100ms reporter wouldn't work without this because data is not harvested). However, if we don't shutdown Delta producer, modifying this value requires thread-safety. Maybe I should just patch opencensus? (it is not protected now)

Reply:
nit: if we remove this call, we could probably remove the config altogether? Seems this is the only caller.

Reply:
The default is 5sec - https://github.com/ray-project/ray/blob/e9755d87a641b02daf82604ef23ce09647158cd4/src/ray/stats/metric.h#L95

And if the comment is true (harvest interval needs to be smaller than reporter interval, otherwise empty collection), we shouldn't remove this? 

Why this value needs to be modified btw?

Reply:
How about we only set it the first time, can remember that in a static variable.